cpp/include/raft/core/cusparse_macros.hpp

```diff
@@ -23,18 +23,6 @@
 #define _CUSPARSE_ERR_TO_STR(err) \
   case err: return #err;
 
-// Notes:
-//(1.) CUDA_VER_10_1_UP aggregates all the CUDA version selection logic;
-//(2.) to enforce a lower version,
-//
-//`#define CUDA_ENFORCE_LOWER
-// #include <raft/sparse/detail/cusparse_wrappers.h>`
-//
-// (i.e., before including this header)
-//
-#define CUDA_VER_10_1_UP (CUDART_VERSION >= 10010)
-#define CUDA_VER_12_4_UP (CUDART_VERSION >= 12040)
```

Comment:
Missed this before. Though noticed on CI. Should we keep the `CUDA_VER_12_4_UP` checks given we support CUDA 12.0-12.9?

Reply:
I need to redo the removal of the checks anyways, but many of them have been there since CUDA 9 (and lots of files are about to be removed from RAFT) so I'm less concerned with those. Given code freeze is looming, let me get the docs and direct code mentions cleaned up at least.